[
    {
        "content": "<p>My notebook was running just fine about an hour ago. After I restarted the kernel, the PBSCluster has the following error message:</p>\n<p>RuntimeError: Command exited with non-zero exit code.<br>\nExit code: 32<br>\nCommand:<br>\nqsub /glade/scratch/huili7/tmp3ox3ei4v.sh<br>\nqsub: request rejected as filter hook 'wallclock' encountered an exception. Please inform Admin.</p>\n<p>I'm not sure what's going on. Anyone else seeing this problem?  Thanks!</p>\n<p>Edit: A similar problem occurs when I try to spawn a server on cheyenne: \" Spawn failed: qsub: request rejected as filter hook 'wallclock' encountered an exception. Please inform Admin\". <span aria-label=\"thinking\" class=\"emoji emoji-1f914\" role=\"img\" title=\"thinking\">:thinking:</span></p>",
        "id": 41536,
        "sender_full_name": "Hui Li",
        "timestamp": 1630440063
    }
]